integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLifecycleSampleScripts.java

```diff
@@ -0,0 +1,448 @@
+      // set the baseName to domain name in create-pv-pvc-inputs.yaml
+      replaceStringInFile(Paths.get(pvpvcBase.toString(), "create-pv-pvc-inputs.yaml").toString(),
+          "baseName: weblogic-sample", "baseName: " + domainName + "-weblogic-sample");
+      // set the pv storage policy to Recycle in create-pv-pvc-inputs.yaml
+      replaceStringInFile(Paths.get(pvpvcBase.toString(), "create-pv-pvc-inputs.yaml").toString(),
+          "weblogicDomainStorageReclaimPolicy: Retain", "weblogicDomainStorageReclaimPolicy: Recycle");
+    });
+
+    // generate the create-pv-pvc-inputs.yaml
+    CommandParams params = new CommandParams().defaults();
+    params.command("sh "
+        + Paths.get(pvpvcBase.toString(), "create-pv-pvc.sh").toString()
+        + " -i " + Paths.get(pvpvcBase.toString(), "create-pv-pvc-inputs.yaml").toString()
+        + " -o "
+        + Paths.get(pvpvcBase.toString()));
+
+    boolean result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create create-pv-pvc-inputs.yaml");
+
+    //create pv and pvc
+    params = new CommandParams().defaults();
+    params.command("kubectl create -f " + Paths.get(pvpvcBase.toString(),
+        "pv-pvcs/" + domainName + "-weblogic-sample-pv.yaml").toString());
+    result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create pv");
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pv {0} to be ready, "
+                + "(elapsed time {1}ms, remaining time {2}ms)",
+                pvName,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvExists(pvName, null),
+            String.format("pvExists failed with ApiException for pv %s",
+                pvName)));
+
+    params = new CommandParams().defaults();
+    params.command("kubectl create -f " + Paths.get(pvpvcBase.toString(),
+        "pv-pvcs/" + domainName + "-weblogic-sample-pvc.yaml").toString());
+    result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create pvc");
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pv {0} to be ready in namespace {1} "
+                + "(elapsed time {2}ms, remaining time {3}ms)",
+                pvcName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> pvcExists(pvcName, domainNamespace),
+            String.format("pvcExists failed with ApiException for pvc %s",
+                pvcName)));
+
+  }
+
+  /**
+   * Delete the persistent volumes since the pv is not decorated with label.
+   */
+  @AfterAll
+  public void tearDownAll() {
+    deleteDomain();
+    for (String domainName : new String[]{"domain1", "domain2"}) {
```

Comment:
There is only 1 domain created in this test

Reply:
Fixed.